source/fundamentals/logging.txt

```diff
@@ -33,7 +33,10 @@ You will learn how to:
 
 .. Who should read this? Where should I go if this isn't the page I was looking for?
 
-.. Not Applicable. No natural redirect location.
+This guide shows how to record events the driver performs.
```

Comment:
TODO: I think this needs to be re-worded.

Reply:
Thanks for the note. I agree "record events the driver performs" is incorrect (I don't think you can perform an event). I rephrased this to "record events in the driver".

Fix: https://github.com/mongodb/docs-java/pull/88/commits/f74439b83566b5873b5358bb1dca34928cc13530

Reply:
Sorry, that TODO was for me to come back to in order to suggest new wording.  But I like what you came up with!